AmazonChimeSDKDemo/AmazonChimeSDKDemo/VideoTileCell.swift

```diff
@@ -24,11 +24,17 @@ class VideoTileCell: UICollectionViewCell {
 
     weak var delegate: VideoTileCellDelegate?
 
-    func updateCell(name: String, isSelf: Bool, isVideoActive: Bool, tag: Int) {
+    func updateCell(name: String, isSelf: Bool, videoTileState: VideoTileState?, tag: Int) {
+        let isVideoActive = videoTileState != nil
+        let isVideoPausedByUser = isVideoActive && videoTileState?.pauseState == .pausedByUserRequest
+
         attendeeName.text = name
         backgroundColor = .systemGray
         isHidden = false
 
+        // Clean up old video image to prevent frame flicker
+        videoRenderView.renderFrame(frame: nil)
```

Comment:
This should be in `prepareForReuse()` instead

Reply:
Good point, moved.